src/app/components/treetable/treetable.ts

```diff
@@ -439,11 +439,15 @@ export class TreeTable implements AfterContentInit {
         }
         return false;
     }
+    
+    toggleNode(node: TreeNode, isExpanded?: boolean) {
```

Comment:
it would be nice to have explicit `expand` and `collapse` methods in addition to toggling